instrumentation/mongo/mongo-common/src/main/java/io/opentelemetry/javaagent/instrumentation/mongo/MongoClientTracer.java

```diff
@@ -130,10 +133,35 @@ private static BsonValue scrub(BsonValue origin) {
     return scrubbed;
   }
 
+  private static final Set<String> COMMANDS_WITH_COLLECTION_NAME_AS_VALUE =
+      new HashSet<>(
+          asList(
+              "aggregate",
+              "count",
+              "distinct",
+              "mapReduce",
+              "geoSearch",
+              "delete",
+              "find",
+              "killCursors",
+              "findAndModify",
+              "insert",
+              "update",
+              "listIndexes"));
```

Comment:
does it make sense to include `create` in this list?

Reply:
I considered it.  It doesn't do any harm, but I left it out because in MongoDB AFAIK it's not so common to create collections explicitly, as MongoDB will create them implicitly on first insert.

But happy to add it (and `drop` as well, I guess) if there's support.

Reply:
sure, let's include them both 👍

Reply:
Added `create`, `drop`, and `createIndexes`